src/Microsoft.AspNet.Razor.Runtime.Precompilation/project.json

```diff
@@ -0,0 +1,40 @@
+{
+    "description": "Supports tag helper resolution during precompilation.",
+    "version": "4.0.0-*",
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/aspnet/razor"
+    },
+    "compilationOptions": {
+        "warningsAsErrors": true
+    },
+    "dependencies": {
+        "Microsoft.Framework.NotNullAttribute.Sources": {
```

Comment:
Seriously @madskristensen this formatting is killing us. Can we fix it so that this doesn't happen by default? I can stop by tomorrow an explain what I think the behavior should be.